src/transac/components/components.module.coffee

```diff
@@ -0,0 +1,10 @@
+###
+#   @desc The Components module is the container reference for all reusable modules. Modules added as depenedencies here should be decoupled.
+###
+angular.module('transac.components',
```

Comment:
I think that on the paper it is a good idea, in the reality for 4 modules it is an overkill. You should simplify this.

If Transac! codebase keeps growing then we will apply this pattern. But if it is not needed it seems to me it is adding useless complexity.